term_posix.go

```diff
@@ -83,6 +84,53 @@ func (t *Term) setSpeed(baud int) error {
 	return termios.Tcsetattr(uintptr(t.fd), termios.TCSANOW, (*syscall.Termios)(&a))
 }
 
+func clamp(v, lo, hi int64) int64 {
+	if v < lo {
+		return lo
+	}
+	if v > hi {
+		return hi
+	}
+	return v
+}
+
+// timeoutVals converts d into values suitable for termios VMIN and VTIME ctrl chars
+func timeoutVals(d time.Duration) (vmin, vtime uint8) {
```

Comment:
I would prefer if you didn't used named return values and naked returns

if d > 0 {
    // compute vtime
    return 0, vtime
} 
return 1, 0